dashboard/app/assets/stylesheets/application.scss

```diff
@@ -538,6 +539,22 @@ $default-modal-width: 640px;
       float: left;
       position: relative;
     }
+
+    .hint {
+      padding: 15px 15px 15px 40px;
+      background-color: $hint-bgcolor;
+      border-radius: 10px;
+    }
+    .hintInner {
```

Comment:
how dramatic of a visual difference would it be if we just removed these rules entirely?